site/docs/selection/clear.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: docs
+menu: docs
+title: Clearing a selection
+permalink: /docs/clear.html
+---
+
+The `clear` selection transformation resets the visualization to its initial configuration:
+
+- For `single` and `multi` selections, it will clear all selected values.
+- For `interval` selections, it will reset the grid position to its initial configuration.
```

Comment:
1) Does this mean there  is no way to "empty" the interval selection if `init` is set? 

2) If 1 is a desired behavior, interval seems a bit inconsistent with single/multi? What if single and multi has "init", shouldn't it reset to the init too?

Reply:
Given the property name is "clear", I think it should really clear the selection rather than "resetting". 

If resetting is a good behavior to support, we could consider a separate `"reset"` property.  

(For now, I think clear is more critical.)

Reply:
I updated the TSDocs and Vega-lite docs, let me know if things are still worded poorly.